web3/src/commonTest/kotlin/dev.icerock.moko.web3/Web3Test.kt

```diff
@@ -57,14 +68,38 @@ private fun createTestAbi(json: Json) = json.parseToJsonElement(testAbiRaw).json
 
 class Web3Test {
     private val infuraUrl = "https://rinkeby.infura.io/v3/5a3d2c30cf72450c9e13b0570a737b62"
-    private val httpClient: HttpClient = HttpClient() {
-        install(Logging){
-            logger = object :Logger{
-                override fun log(message: String) {
-                    println(message)
+
+    // When new test added, it should be tested with this real client,
+    // And then mocked request should be added to client below,
+    // since networks are unstable (especially test networks)
+//    private val httpClient: HttpClient = HttpClient {
+//        install(Logging){
+//            logger = object : Logger {
+//                override fun log(message: String) {
+//                    println(message)
+//                }
+//            }
+//            level = LogLevel.ALL
+//        }
+//    }
+    private val httpClient: HttpClient = HttpClient(MockEngine) {
```

Comment:
extract creation of mocked engine to separated function like in my branch.
also extract json string into separated file with mock data - please not mix code and mock strings.